snuba/pipeline/pipeline_delegator.py

```diff
@@ -114,6 +118,6 @@ def build_planner(
     ) -> QueryPlanner[ClickhouseQueryPlan]:
         # For composite queries, we just build the primary pipeline / query plan;
         # running multiple concurrent composite queries is not currently supported.
-        primary_builder_id, _others = self.__selector_func(query)
+        primary_builder_id, _others = self.__selector_func(query, "")
```

Comment:
Why does this one not get a referer?

Reply:
Unfortunately I didn't have an easy way to get hold of the referrer here. I'm not really sure why the differences in building the plan vs the execution pipeline exist. `build_execution_pipeline` gets the entire request while `build_planner` gets only the query, but I decided not to mess with it here.